index.html

```diff
@@ -1,14 +1,31 @@
-<!DOCTYPE html>
-<html>
+<!doctype html>
+<html lang="en">
+
 <head>
-    <meta charset='utf-8'>
-    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
-    <title>Page Title</title>
-    <meta name='viewport' content='width=device-width, initial-scale=1'>
-    <link rel='stylesheet' type='text/css' media='screen' href='main.css'>
-    <script src='main.js'></script>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Studio Lenzing</title>
+  <!-- Bootstrap CSS -->
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+    integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  <!-- Google Font - Open Sans -->
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700&display=swap" rel="stylesheet">
+  <!-- Custom CSS -->
+  <link rel="stylesheet" type="text/css" href="assets/studio_lenzing.css">
 </head>
+
 <body>
-    
+  <nav class="topnav d-flex align-items-center justify-content-between">
+    <div class="logo-container">
+      <a class="logo-title" href="#home">Studio Lenzing</a>
+    </div>
+      <ul class="d-flex list-unstyled my-auto text-uppercase">
```

Comment:
Fix spacing

Reply:
There are VS Code extensions you can use for formatting files :D

Reply:
I used VSCode's own function for fomatting sa spacing, "Format Document".
Do you have any recommendations for this?